apps/token-manager/app/src/App.js

```diff
@@ -181,13 +216,11 @@ export default observe(
           tokenDecimals: parseInt(tokenDecimals, 10),
           tokenSupply: parseInt(tokenSupply, 10),
         },
-        holders: holders
-          ? holders
-              .map(holder => ({ ...holder, balance: new BN(holder.balance) }))
-              .sort((a, b) => b.balance.cmp(a.balance))
-          : [],
+        holders: updatedHolders,
         tokenDecimals: new BN(tokenDecimals),
         tokenSupply: new BN(tokenSupply),
+        maxAccountTokens: new BN(maxAccountTokens),
+        groupMode: hasEveryHolderSameBalance(updatedHolders, tokenDecimalsBase),
```

Comment:
@sohkai @izqui What do you think about replacing this by `maxAccountTokens === '1'`?

Reply:
Yeah, I think it could just be `tokenTransfersEnabled && maxAccountTokens === '1'` (or the equivalent check of `maxAccountTokens.eq(tokenDecimalsBase)`).

That way the UI also wouldn't change when new democracy orgs mint new tokens.